tests/setup.js

```diff
@@ -0,0 +1,59 @@
+// Load test helpers
+jest.dontMock('underscore');
+jest.dontMock('jquery');
+
+global.OO = { publicApi: {}, platform: 'MacIntel', os: {}, browser: { version:1, webkit:true }, TEST_TEST_TEST: true};
+global.OO.getRandomString = function() { return Math.random().toString(36).substring(7); };
+
+// The function setTimeout from jsdom is not working, this overwrites the function with the function defined by node
+global.window.setTimeout = setTimeout;
+global.window.setInterval = setInterval;
+global.window.clearInterval = clearInterval;
+global.navigator = window.navigator;
+global.window.$ = require("jquery");
+OO.$ = global.window.$;
+
+global.expect = require('expect.js');
+
+// a wrapper domparser simulating Mozilla DOMParser in the browser:
+window.DOMParser = function() {};
+
+require.requireActual("../html5-common/js/utils/InitModules/InitOOUnderscore.js");
+
+OO._.extend(window.DOMParser.prototype, {
```

Comment:
Would it be better to use `Object.assign`, `Array.forEach`, etc., in order to minimize dependency on `OO` and underscore?

Reply:
Yep, will change. I copied this from test_lib.js and didn't notice. Nice catch!